pkg/actuators/machine/utils.go

```diff
@@ -51,17 +51,38 @@ func getRunningInstance(machine *machinev1.Machine, client awsclient.Client) (*e
 // getRunningInstances returns all running instances that have a tag matching our machine name,
 // and cluster ID.
 func getRunningInstances(machine *machinev1.Machine, client awsclient.Client) ([]*ec2.Instance, error) {
-	runningInstanceStateFilter := []*string{aws.String(ec2.InstanceStateNameRunning), aws.String(ec2.InstanceStateNamePending)}
+	runningInstanceStateFilter := []*string{aws.String(ec2.InstanceStateNameRunning)}
```

Comment:
I believe this will help with e2e flake: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_cluster-api-provider-aws/218/ci-kubernetes-aws-actuator/885/

It seems that the instance is not coming up within test timeouts.  This will ensure we wait for the instance to actually be running instead of pending.

Reply:
> It seems that the instance is not coming up within test timeouts. This will ensure we wait for the instance to actually be running instead of pending.

If that's the case then I think this change should be its own discrete PR.